Mathlib/Topology/Algebra/ContinuousMonoidHom.lean

```diff
@@ -415,6 +413,9 @@ namespace PontryaginDual
 
 open ContinuousMonoidHom
 
+noncomputable instance : FunLike (PontryaginDual A) A circle :=
```

Comment:
Is this necessary?

Reply:
This doesn’t have to be `noncomputable` but it cannot find the instance otherwise